cal_ratio_trainer/config.py

```diff
@@ -372,4 +392,5 @@ class BuildMainTrainingConfig(BaseModel):
     ConvertDiVertAnalysisConfig: "default_divert_config",
     BuildMainTrainingConfig: "default_build_main_training_config",
     ConvertxAODConfig: "default_xaod_config",
+    ScorePickleConfig: "default_score_pickle_config",
```

Comment:
I'm not seeing the `default_score_pickle_config.yaml` file... how have you been able to run this without that default file being present in the repo?